core/keyserver/keyserver.go

```diff
@@ -132,7 +136,7 @@ func (s *Server) getEntry(ctx context.Context, userID, appID string, firstTreeSi
 			LogId: s.logID,
 		})
 	if err != nil {
-		log.Printf("tlog.GetLatestSignedLogRoot(%v): %v", s.logID, err)
+		glog.Warningf("tlog.GetLatestSignedLogRoot(%v): %v", s.logID, err)
```

Comment:
When I check logs, I first look for errors (not warning). An invalid user input would make the API call fail, and I think it should be an error. What do you think? If you thing these are fine as warnings, that's works for me.